opl/investigator/elasticsearch_loader.py

```diff
@@ -21,7 +26,7 @@ def load(server, index, query, paths):
         f"Querying ES with url={url}, headers={headers} and json={json.dumps(data)}"
     )
 
-    response = opl.http.get(url, headers=headers, json=data)
+    response = opl.http.get(url, auth=HTTPBasicAuth(open_search_username, open_search_password), headers=headers, json=data)
```

Comment:
This is not backwards compatible. We will not be able to migrate all our tests at one moment. We need the change that will not break current setup.